addons/pinniped/config-controller/hack/run.sh

```diff
@@ -5,22 +5,142 @@
 
 set -euo pipefail
 
-MANIFESTS_DIR="./manifests"
-RBAC_FILE="${MANIFESTS_DIR}/role.yaml"
-DEPLOYMENT_FILE="${MANIFESTS_DIR}/deployment.yaml"
+YELLOW='\033[0;33m'
+DEFAULT='\033[0m'
+
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TF_ROOT="${MY_DIR}/../../../.."
```

Comment:
Just a suggestion in case things ever move around, but may want to change this to `$(git rev-parse --show-toplevel)` so we don't have to rely on folder structure. But this is good!